frontend/javascripts/oxalis/api/api_latest.ts

```diff
@@ -747,9 +747,18 @@ class TracingApi {
   }
 
   /**
-   * Returns the length of the shortest path between two nodes in nanometer and in voxels.
-   */
-  measurePathLengthBetweenNodes(sourceNodeId: number, targetNodeId: number): [number, number] {
+   * Returns the length of the shortest path between two nodes in nanometer and in voxels
+   * as well as the shortest path between two nodes as an array of node IDs.
+   * Note that the array is ordered from target to source node.
+   */
+  measurePathLengthBetweenNodes(
```

Comment:
this is a backwards-incompatible change right now. I'd rename this function and add another `measurePathLengthBetweenNodes` function which delegates to this implementation and simply extracts the lengthVx and lengthNm parameter to return it.